### PR TITLE
fix(DB/reference_loot_template): Corrected Shadow Labyrinth gem drops

### PR DESCRIPTION
**Pull request by samsam1993**
## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

Heroic Shadow Labyrinth bosses dropped incorrect gem. They had a chance to drop Glinting Fire Opal, but this line of SQL corrects them to drop Etched Fire Opal instead, as they should.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/17156
- Closes https://github.com/chromiecraft/chromiecraft/issues/6092

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/SL_gem_loot_fix.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+UPDATE `acore_world`.`reference_loot_template` SET `Item` = '30559', `Comment` = 'Etched Fire Opal' WHERE (`Entry` = '43010') and (`Item` = '30556');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/SL_gem_loot_fix.sql:2]: remove `acore_world` and the quotes.

see https://www.azerothcore.org/wiki/sql-standards for SQL standards

File: data/sql/updates/pending_db_world/SL_gem_loot_fix.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/SL_gem_loot_fix.sql:1]:
```suggestion
--
```

File: data/sql/updates/pending_db_world/SL_gem_loot_fix.sql (hunk 1 above)
Comment (elthehablo) [inline data/sql/updates/pending_db_world/SL_gem_loot_fix.sql:2]:
```suggestion
UPDATE `reference_loot_template` SET `Item` = '30559', `Comment` = 'Etched Fire Opal' WHERE `Entry` = 43010 AND `Item` = 30556;
```

**Review by elthehablo — APPROVED** [review]

**Comment by elthehablo** [comment]
went ahead and commited some changes as this PR seemed dead. sorry

**Pull request merged**